locales/de-de.json

```diff
@@ -0,0 +1,74 @@
+{
+	"select-user-type.who_you_are": "Was du bist? (Wähle jetzt aus, du kannst es später ändern)",
+	"select-user-type.driver": "Fahrer",
+	"select-user-type.passenger": "Passagier",
+	"request-phone.type_your_phone": "Bitte sende uns deine Telefonnummer. Wir geben diese Nummer nicht ohne Erlaubnis weiter.",
+	"request-phone.button_text": "Nummer senden",
+	"request-phone.all_set": "👌 Alles klar soweit, du kannst jetzt eine Fahrt bestellen!",
+	"passenger-index.choose_taxi_type": "Welche Art von Taxi willst du?",
+	"passenger-index.car": "🚗 Auto",
+	"passenger-index.motorbike": "🏍 Roller / Motorrad",
+	"passenger-index.s": "🔧 Einstellungen",
+	"passenger-index.error_only_known_type": "Unbekannte Auswahl, bitte wähle das Zweirad, Auto oder die Einstellungen aus.",
```

Comment:
Since this is an error message, I don't think it's necessary to explicitly state "unknown choice". I would just say:

Bitte wähle Roller, Auto oder Einstellungen aus.

@ro31337 Does the UI make clear it's displaying an error message (color, etc. ?)